.eslintrc.json

```diff
@@ -16,6 +16,13 @@
       "parserOptions": {"ecmaVersion": 9, "ecmaFeatures": {"jsx": true}},
       "globals": {},
       "rules": {"no-unused-vars": ["off"]}
+    },
+    {
+      "files": ["test/**/*.mjs"],
+      "env": {"es6": true, "node": true},
+      "rules": {
+        "max-len": ["off"]
+      }
```

Comment:
This should only be needed until https://github.com/sanctuary-js/sanctuary-style/pull/22 lands